new-pipeline/evo.html

```diff
@@ -4,17 +4,17 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    
+
```

Comment:
There are whitespace changes in this file that we don't need.
Please drop these changes.

Reply:
Yeah I will.